api/openapi-spec/swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Kubernetes",
-   "version": "v1.7.0"
+   "version": "v1.5.0"
```

Comment:
Oops. I dont know how that happened. I just ran the script under /hack. Ill re run it again.